docs/platforms/gke/base/use-cases/inference-ref-arch/inference-best-practices.md

```diff
@@ -0,0 +1,193 @@
+
+- Implement robust traffic routing mechanisms and automated failover procedures
+  to ensure continuous service availability. For more information, see
+  [Exposing inference workloads and global load balancing](#exposing-inference-workloads).
+- Minimize inter-region network traffic by making model files for your inference
+  workloads available across different regions. For example, you can:
+
+  - Store model files in a
+    [dual- or multi-region bucket](https://cloud.google.com/storage/docs/locations).
+    When model files are stored in dual- or multi-region buckets, we recommend
+    that you enable
+    [Anywhere Cache](https://cloud.google.com/storage/docs/anywhere-cache) to
+    help you speed up scaling up your inference workloads, and to help optimize
+    Cloud Storage costs.
+  - Store model files in a single-region bucket, and replicate its data across
+    regions using
+    [Storage Transfer Service](https://cloud.google.com/storage-transfer/docs/overview).
+  - Store model files in single-region buckets, and run at least one instance of
+    the Model Downloader for each region to download model files for inference
+    workloads running in that region.
+
+## Online inference with GPUs design choices
+
+This section focuses on the design choices that apply to the
+[Online inference with GPUs](/platforms/gke/base/use-cases/inference-ref-arch/examples/online-inference-gpu/README.md)
+use case. The Online inference with GPUs use case builds on top of the GKE
+Inference reference architecture. The design choices described in this section
+apply to the Online inference with GPUs use case, in addition to the
+[GKE Inference reference architecture design choices](#gke-inference-reference-architecture-design-choices).
+
+The Online inference with GPUs use case design choices cover the following:
+
+- GPU selection
+- GPUs obtainability
+
+### GPU selection
+
+To implement the Online inference with GPUs use case, we selected GPUs by
+considering the following selection criteria:
+
+- Obtainability
+- Cost-effectiveness
+- Performance
+
+For the Online inference with GPUs, we selected the following GPUs:
+
+- **NVIDIA** **L4 GPUs (G2 machines)**: Ideal for cost-effective,
+  high-throughput online inference scenarios, offering a good balance of
+  performance and efficiency.
+- **NVIDIA H100 (A3 machines)**: Selected for demanding inference workloads
+  where maximum throughput and low latency are critical.
+- **NVIDIA H200 (A3 machines)**: Selected for providing unparalleled performance
+  for the most memory-intensive inference workloads.
+
+The following table provides a general overview of model compatibility and
+recommended machine types, for the
+[models that are in scope as examples](#ai-models-in-scope-as-examples):
+
+| GPU  | Compatible Models               | Machine type                                                                                                 | GPU count                                               |
```

Comment:
Reminder to add models mentioned in https://github.com/GoogleCloudPlatform/accelerated-platforms/pull/249